openmdao/docs/features/building_blocks/solvers/nonlinear/nonlinear_block_gs.rst

```diff
@@ -38,6 +38,14 @@ NonlinearBlockGS Options
     NonlinearBlockGS
     options
 
+
+Aitken relaxation
```

Comment:
Not sure it's worth another commit and CI run to capitalize the title "Relaxation".  Probably not. Checked the link, it works.